### PR TITLE
[ARXIVCD-608] missing file to log in JSON

### PR DESCRIPTION
**Pull request by ntai-arxiv**

### CODE REVIEW

File: script/sync_prod_to_gcp/stanza/plugins/arxiv_sync2gcp_log.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ parameters:
     label: Log Path
     description: Path to the sync to GCP log
     type: string
-    default: "/opt_arviv/logs/sync/*"
+    default: "/opt_arviv/e-prints/logs/sync/*"
```
Comment (bmaltzan) [inline script/sync_prod_to_gcp/stanza/plugins/arxiv_sync2gcp_log.yaml:11]: /opt_arviv ->
/opt_arxiv

File: script/sync_prod_to_gcp/README.md [hunk 2]
```diff
@@ -2,14 +2,102 @@
...
+
+There are two logging, one plain text, and the second is NDJSON that is designed for GCP. The JSON logging goes into
+`/opt_arxiv/e-prints/logs/sync` and Stanza sends it out to GCP. See JSON logging.
+
+### Stanza plug-in for sync
+
+The plugin must be deployed on "arxiv-sync" host.
+
+`arxiv-browse/scripts/sync_prod_to_gcp/stanza/plugins/arxiv_sync2gcp_log.yaml` -> `/opt/observeiq/stanza/plugins/arxiv_sync2gcp_log.yaml`
+
```
Comment (bmaltzan) [inline script/sync_prod_to_gcp/README.md:89]: /opt/observiq

**Review by bdc34 — APPROVED** [review]

**Comment by bmaltzan** [comment]
Stanza sends log files for many apps like browse and search, 
we might want to use a repo for devops, like:
https://github.com/arXiv/arxiv-operations/tree/master/stanza

Should our repos with config details be public?

File: script/sync_prod_to_gcp/stanza/plugins/arxiv_sync2gcp_log.yaml (hunk 1 above)
Comment (ntai-arxiv) [reply script/sync_prod_to_gcp/stanza/plugins/arxiv_sync2gcp_log.yaml:11]: oops.

**Comment by ntai-arxiv** [comment]
> Stanza sends log files for many apps like browse and search, we might want to use a repo for devops, like: https://github.com/arXiv/arxiv-operations/tree/master/stanza

First, the stanza configs/plugins should be in somewhere. Yes to arxiv-ops. And probably things like cron jobs as well in git as the source of (semi) truth.  We'd then need a mechanism to automate the deployment. 

> Should our repos with config details be public?

I'm not sure what this means. Public repo?

**Pull request merged**